tests/gtest/avif_fuzztest_helpers.h

```diff
@@ -281,13 +281,18 @@ inline auto ArbitraryBaseAvifDecoder() {
 
 #if defined(AVIF_ENABLE_EXPERIMENTAL_GAIN_MAP)
 // Generator for an arbitrary DecoderPtr with base options and gain map
-// options fuzzed, with the exception of 'ignoreColorAndAlpha' (because it would
-// break most tests' assumptions).
+// options fuzzed.
 inline auto ArbitraryAvifDecoderWithGainMapOptions() {
+  // Always decode at least the color image, since most tests
+  // assume that if the file/buffer is successfully decoded.
```

Comment:
This comment is still accurate even though we now always decode the color and alpha images. Not sure if we should update this comment.